src/plugins/minimal_scene/MinimalScene.hh

```diff
@@ -380,101 +380,101 @@ namespace gz::gui::plugins
     /// \param[in] _view_controller The camera view controller type to set
     public: void SetCameraViewController(const std::string &_view_controller);
 
-    /// \brief Slot called when thread is ready to be started
-    public Q_SLOTS: void Ready();
+  /// \brief Slot called when thread is ready to be started
+  public Q_SLOTS: void Ready();
```

Comment:
undo indentation? looks like the original indentation is correct?

Reply:
I think that the indentation may be incorrect on `main`, so I have fixed it here.

Reply:
(Just now, so take another look)